torch/nn/_intrinsic/qat/modules/conv_bn_relu.py

```diff
@@ -0,0 +1,184 @@
+        self.momentum = momentum
+        self.freeze_bn = freeze_bn
+        self.num_features = out_channels
+        self.gamma = Parameter(torch.Tensor(out_channels))
+        self.beta = Parameter(torch.Tensor(out_channels))
+        self.register_buffer('running_mean', torch.zeros(out_channels))
+        self.register_buffer('running_var', torch.ones(out_channels))
+        self.register_buffer('num_batches_tracked', torch.tensor(0, dtype=torch.long))
+        self.observer = activation_fake_quant()
+        self.weight_fake_quant = weight_fake_quant()
+
+    def reset_running_stats(self):
+        self.running_mean.zero_()
+        self.running_var.fill_(1)
+        self.num_batches_tracked.zero_()
+
+    def reset_parameters(self):
+        self.reset_running_stats()
+        if self.affine:
+            init.uniform_(self.gamma)
+            init.zeros_(self.beta)
+
+    def _forward(self, input):
+        # exponential_average_factor is self.momentum set to
+        # (when it is available) only so that if gets updated
+        # in ONNX graph when this node is exported to ONNX.
+        if self.momentum is None:
+            exponential_average_factor = 0.0
+        else:
+            exponential_average_factor = self.momentum
+
+        if self.training and self.track_running_stats:
+            # TODO: if statement only here to tell the jit to skip emitting this when it is None
+            if self.num_batches_tracked is not None:
+                self.num_batches_tracked += 1
+                if self.momentum is None:  # use cumulative moving average
+                    exponential_average_factor = 1.0 / float(self.num_batches_tracked)
+                else:  # use exponential moving average
+                    exponential_average_factor = self.momentum
+        scaled_weight = self.gamma * self.weight / torch.sqrt(self.running_var)
+        conv = conv2d_forward(input, self.padding_mode, self.padding,
+                              self.weight_fake_quant(scaled_weight), self.bias,
+                              self.stride, self.dilation, self.groups)
+        batch_mean = torch.mean(conv, dim=[0, 2, 3])
+        n = input.numel() / self.num_features
+        batch_var = torch.var(conv, dim=[0, 2, 3]) * (n / (n - 1))
+        self.running_mean = exponential_average_factor * batch_mean + (1 - exponential_average_factor) * self.running_mean
+        self.running_var = exponential_average_factor * batch_var + (1 - exponential_average_factor) * self.running_var
+
+        if not self.freeze_bn:
+            conv *= torch.sqrt(self.running_var) / torch.sqrt(batch_var)
+            conv += self.beta - self.gamma * (batch_mean / torch.sqrt(batch_var))
+        else:
+            conv += self.beta - self.gamma * self.running_mean / torch.sqrt(self.running_var)
+
+        return conv
+
+    def forward(self, input):
+        return self.observer(self._forward(input))
+
+    @classmethod
+    def from_float(cls, mod, qconfig):
```

Comment:
Ensure that bn module has affine=True and tracking_running_stats= True

Reply:
I think we can ensure that in conversion